scripts/build/ci-build/Dockerfile

```diff
@@ -100,7 +100,11 @@ RUN tar xf cue_${CUE_VERSION}_Linux_x86_64.tar.gz -C /tmp cue
 # Use old Debian (this has support into 2022) in order to ensure binary compatibility with older glibc's.
 FROM debian:stretch-20210208
 
+<<<<<<< Updated upstream
 ENV GOVERSION=1.16 \
+=======
+ENV GOVERSION=1.16.0 \
+>>>>>>> Stashed changes
```

Comment:
something funky here

Reply:
@ryantxu thanks for noticing, removed it.